nestkernel/connector_base.h

```diff
@@ -132,12 +133,14 @@ class ConnectorBase
   virtual void get_connections( size_t source_gid,
     size_t thrd,
     synindex synapse_id,
+    long_t label,
```

Comment:
For symmetry with `synapse_id` and usage elsewhere, shouldn't this be `synapse_label` instead of just `label`? Applies to all overriding declarations, too.